### PR TITLE
Update back-up-email.md

### PR DESCRIPTION
**Pull request by ninobb**
I think we really need to clarify that Archiving is NOT backup. In no way is Archiving similar to backup, unless we are talking about storing older stuff.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/back-up-email.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ For more information about how admins can recover deleted items in Exchange Onli
 - [Change how long permanently deleted items are kept for an Exchange Online mailbox](recipients-in-exchange-online/manage-user-mailboxes/change-deleted-item-retention.md).
 
 > [!NOTE]
-> Point in time restoration of mailbox items is out of scope for the Exchange Online service. However, Exchange Online offers great retention and recovery support for your organization's email infrastructure, and your mailbox data is available when you need it, no matter what happens. For more information about additional options, see the following topics:
+> Point in time restoration of mailbox items is out of scope for the Exchange Online service (there might be 3rd party solutions providing that functionality). However, Exchange Online offers great retention and recovery support for your organization's email infrastructure, and your mailbox data is available when you need it, no matter what happens. For more information about additional options, see the following topics:
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/back-up-email.md:48]:
```suggestion
> Point in time restoration of mailbox items is out of scope for the Exchange Online service (there might be third-party solutions providing that functionality). However, Exchange Online offers great retention and recovery support for your organization's email infrastructure, and your mailbox data is available when you need it, no matter what happens. For more information about additional options, see the following topics:
```

File: Exchange/ExchangeOnline/back-up-email.md [hunk 2]
```diff
@@ -55,7 +55,7 @@ For more information about how admins can recover deleted items in Exchange Onli
 
 ## How do users backup Outlook data?
 
-In Exchange Online, the best way to provide a backup for users is with Exchange Online Archiving. Using Outlook to backup data into PST files aren't recommended due to the loss of discoverability and control of the content.
+Exchange Online does not provide a way to perform traditional backup of mailboxes (as this implies the ability to restore a mailbox to a state the mailbox was in when backup was taken). The best way to provide additional storage for user emails is with Exchange Online Archiving. Using Outlook to backup data into PST files aren't recommended due to the loss of discoverability and control of the content.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/back-up-email.md:58]:
```suggestion
Exchange Online does not provide a way to perform a traditional backup of mailboxes (as this implies the ability to restore a mailbox to a state the mailbox was in when the backup was taken). The best way to provide additional storage for user emails is with Exchange Online Archiving. Using Outlook to backup data into PST files aren't recommended due to the loss of discoverability and control of the content.
```

**Comment by scanum** [comment]
@mattpennathe3rd Thank you for submitting feedback and contributing to the docs. We are currently investigating this.

**Pull request closed without merging**